RecoEgamma/EgammaPhotonProducers/src/ConversionProducer.cc

```diff
@@ -11,56 +11,193 @@ Description: Produces converted photon objects using default track collections
 <Notes on implementation>
 */
 //
-// Original Authors:  Hongliang Liu
+// Original Authors:  Hongliang Liu, UC of Riverside US, Nancy Marinelli Univ of Notre Dame
 //         Created:  Thu Mar 13 17:40:48 CDT 2008
 //
 //
 
-// system include files
-#include <memory>
-#include <map>
-
-#include "RecoEgamma/EgammaPhotonProducers/interface/ConversionProducer.h"
+#include "CommonTools/Statistics/interface/ChiSquaredProbability.h"
+#include "DataFormats/CaloRecHit/interface/CaloCluster.h"
+#include "DataFormats/EcalDetId/interface/EcalSubdetector.h"
+#include "DataFormats/EgammaCandidates/interface/Conversion.h"
+#include "DataFormats/EgammaCandidates/interface/Photon.h"
+#include "DataFormats/EgammaReco/interface/BasicCluster.h"
```

Comment:
I wonder if BasicCluster's and SuperCluster's are even needed, since only CaloCluster's seem to be used in the code